pyproject.toml

```
@@ -19,7 +19,7 @@ maintainers = [
 dependencies = [
 	"ansys-api-systemcoupling==0.1.0",
 	"grpcio>=1.30.0",
-	"grpcio-status>=1.30.0,<1.51.2",
+	"grpcio-status>=1.30.0,<1.51.4",
```

Why did you need to change this dependency for migrating to pyansys/actions v4?
```suggestion
	"grpcio-status>=1.30.0,<1.51.2",
```